test/test.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.10.3-alpine as base
```

Comment:
The base image could be a pure alpine without golang.

Reply:
Is that any more efficient? Since i'm running `go get` command, if i don't use `golang:1.10.3-alpine` as base, I'll get a `/bin/sh: go: not found` error. To avoid this, I'd have to have to run `apk add go` anyway and then I'd have to implement additional steps in order to properly configure the go env. Using `golang:1.10.3-alpine` as base addresses these issues with minimal dependencies. If I'm mistaken, please let me know! I appreciate your feedback! 😃

Reply:
ah ... sorry I misunderstood your base image. That said, here are two images, you only need Go in one of them. 
So in this case, `iron/go:latest` should be the image that Go is not required.

Reply:
Sorry, I just shipped it. If there is anything should be modified, we can have another PR for it. :)

Reply:
I changed it from `iron/go` to `alpine` and it only reduced the image size by .3mb. I'll keep working on it though.